src/test/java/MyArrayListTest.java

```diff
@@ -0,0 +1,60 @@
+import com.homework.model.MyArrayList;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class MyArrayListTest {
```

Comment:
as for me using additional array just complicate logic of testing
but you can not change it, the main thing is that you understand how it works